home/ivp-planning/user-interface/settings-panel.md

```diff
@@ -5,9 +5,12 @@ The settings panel offers a variety of options to customize iVP. It is divided i
 * [Global settings:](../settings/global-settings.md) user interface scale, globally used unit, floor plan settings, etc.
```

Comment:
I think we should remove this page here and add the information on how to open the panel to Intelligent Virtual Platform/iVP Planning/Settings. The description and image need to be updated because the cog wheel button to open the panel has been removed. The bullet list with the individual settings should be replaced with "page links" like the one in the [changelog](https://ixtenda.gitbook.io/help-ivp-intelligent-virtual-platform/home/ivp-planning/changelog).